pkgs/build-support/rust/hooks/cargo-build-hook.sh

```diff
@@ -43,7 +43,8 @@ cargoBuildHook() {
         ${cargoBuildProfileFlag} \
         ${cargoBuildNoDefaultFeaturesFlag} \
         ${cargoBuildFeaturesFlag} \
-        ${cargoBuildFlags}
+        ${cargoBuildFlags} \
+        ${cargoBuildFlagsArray[@]}
```

Comment:
doesn't this have to be

```suggestion
        "${cargoBuildFlagsArray[@]}"
```

?

and for consistency's sake, perhaps it's better to do

```suggestion
        ${cargoBuildFlagsArray+"${cargoBuildFlagsArray[@]}"}
```

Reply:
Oh you're right, good catch.